src/components/instructor_files/instructor_files.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <div id="instructor-files-component" v-if="project !== null">
+    <FileUpload ref="instructor_files_upload"
+                @upload_files="add_instructor_files($event)">
+    </FileUpload>
+
+    <div id="instructor-files-label"> Uploaded Instructor Files
+      <div class="collapse-button"
+           @click="collapsed = !collapsed">( {{collapsed? 'Show' : 'Collapse'}} )</div>
+    </div>
+
+    <div id="viewing-area">
+      <div id="column-of-files" :style="{display: collapsed? 'none' : 'block'}">
+        <div v-for="instructor_file of instructor_files" :key="instructor_file.pk">
+          <single-instructor-file :file="instructor_file"
+                       @open_file="view_file($event)">
+          </single-instructor-file>
+        </div>
+      </div>
+      <div id="instructor-file-viewer-wrapper"
+           :style="{left: collapsed ? '0' : '390px'}">
+        <MultiFileViewer ref="instructor_files_viewer"
+                         height_of_view_file="600px"
+                         @num_files_viewing_changed="num_files_currently_viewing = $event">
+        </MultiFileViewer>
+        <div v-if="num_files_currently_viewing === 0 && instructor_files.length !== 0"
+             class="helpful-message">
+          Click on a file to view its contents.
```

Comment:
I'm inclined to say that this behavior should happen in the MultiFileViewer component. If we need to customize the message, it can use a slot. Was there something to do with positioning that makes it need to be handled here?